scripts/dev/replace-new-version.sh

```
@@ -0,0 +1,55 @@
+APP_LOG_DIR="/home/ubuntu/app/logs"
+APP_LOG_FILE="$APP_LOG_DIR/application-$(date +%Y%m%d-%H%M%S).log"
+
+log "애플리케이션 로그 파일: $APP_LOG_FILE"
+
+sudo nohup java \
+    -Dspring.profiles.active=dev \
+    -Duser.timezone=Asia/Seoul \
+    -Dserver.port=8080 \
+    -Ddd.service=WellMeet-Backend \
+    -Ddd.env=dev \
+    -jar "$JAR_FILE" > "$APP_LOG_FILE" 2>&1 &
```

Shellcheck (0.10.0)

[warning] sudo doesn't affect redirects. Use ..| sudo tee file

(SC2024)